core/src/main/java/io/questdb/griffin/model/ExecutionModel.java

```diff
@@ -27,10 +27,41 @@
 public interface ExecutionModel {
     int COPY = 5;
     int CREATE_TABLE = 2;
+    int EXPLAIN = 7;
     int INSERT = 4;
+    int MAX = EXPLAIN + 1;
     int QUERY = 1;
     int RENAME_TABLE = 3;
     int UPDATE = 6;
 
     int getModelType();
+
+    default QueryModel getQueryModel() {
+        return null;
+    }
+
+    default CharSequence getTargetTableName() {
```

Comment:
`null` table name is unhandled in plan sinks

Reply:
Fixed.